content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst

```diff
@@ -0,0 +1,102 @@
+under the :guilabel:`Products` heading, enable the :guilabel:`Product Packagings` feature, and click
+:guilabel:`Save`.
+
+.. image:: packaging/enable-packagings.png
+   :align: center
+   :alt: Enable packagings by selecting "Product Packagings".
+
+Create packaging
+================
+
+Packagings can be created directly on the product form, or from the :guilabel:`Product Packagings`
+page.
+
+From product form
+-----------------
+
+Create packagings on a product form by going to :menuselection:`Inventory app --> Products -->
+Products`, and select the desired product.
+
+Under the :guilabel:`Inventory` tab, scroll down to the :guilabel:`Packaging` section, and click
+:guilabel:`Add a line`. In the table, fill out the following fields:
+
+- :guilabel:`Packaging`: (required) name of packaging that appears on sales/purchase orders as a
+  packaging option for the product.
+- :guilabel:`Contained quantity`: (required) amount of product in the packaging.
+- :guilabel:`Unit of Measure`: (required) measurement unit for quantifying the product.
+- :guilabel:`Sales`: check this option for packagings intended for use on sales orders.
+- :guilabel:`Purchase`: check this option for packagings intended for use on purchase orders.
+
+.. note::
+   Access additional fields in the :guilabel:`Packaging` table below by clicking the
+   :guilabel:`(sliders)` icon on the far right, and selecting the desired options.
+
+   .. image:: packaging/slide.png
+      :align: center
+      :alt: Show the additional options menu's icon: sliders.
+
+- :guilabel:`Barcode`: identifier for tracing packaging in stock moves or pickings using the
+  :ref:`Barcode app <barcode/operations/intro>`. Leave blank if not in use.
+- :guilabel:`Company`: indicates the packaging is only available at the selected company. Leave
+  blank to make the packaging available across all companies.
+
+.. example::
+   To create a packaging type for six units of the product, `Grape Soda`, begin by clicking
+   :guilabel:`Add a line`. In the line, name the :guilabel:`Packaging` `6-pack` and set the
+   :guilabel:`Contained quantity` to `6`. Repeat this process for additional packagings.
+
+   .. image:: packaging/create-product-packaging.png
+      :align: center
+      :alt: Create 6-pack case for product.
+
+From product packagings page
+----------------------------
+
+To view all packagings that have been created, go to :menuselection:`Inventory app --> Configuration
+--> Product Packagings`. Doing so reveals the :guilabel:`Product Packagings` page with a complete
+list of all packagings that have been created for all products. Create new packagings by clicking
+:guilabel:`New`.
+
+.. example::
+   Two soda products, `Grape Soda` and `Diet Coke`, have three types of packagings configured. On
+   the :guilabel:`Product Packagings` page, each product can be sold as a `6-Pack` that contains 6
+   products, as a `12-Pack` of 12 products, or as a `Case` of 32 products.
```

Comment:
```suggestion
   the :guilabel:`Product Packagings` page, each product can be sold as a :guilabel:`6-Pack` that
   contains 6 products, as a :guilabel:`12-Pack` that contains 12 products, or as a :guilabel:`Case`
   that contains 32 products.
```

Reply:
used GUIs bc that's how it appears on the screenshot/UI

Reply:
While it's true that we see Grape Soda on the UI,  it's an example product and doesn't appear in every customer database. Compared to something like a field or a header (like "Product Packagings"), I think grape soda is just user input and  gets back ticks `, not a guilabel (: